src/main/java/com/denizenscript/ddiscordbot/objects/DiscordUserTag.java

```diff
@@ -420,6 +420,25 @@ public static void register() {
             }
             return list;
         });
+
+        // <--[tag]
+        // @attribute <DiscordUserTag.is_banned[<group>]>
+        // @returns ElementTag(boolean)
+        // @plugin dDiscordBot
+        // @description
+        // Returns if the user is banned from a certain group.
+        // -->
+        tagProcessor.registerTag(ElementTag.class, DiscordGroupTag.class, "is_banned", (attribute, object, group) -> {
+            boolean isBanned = false;
+            UserSnowflake user = UserSnowflake.fromId(object.user_id);
+            for (Guild.Ban ban : group.getGuild().retrieveBanList().complete()) {

```

Comment:
This strongly seems like the wrong way to implement this. Have you checked for a more direct method?

Reply:
Looks like you can use `Guild#getManager().getBans()`

Reply:
You can use `retrieveBan(User)` to get the `Guild.Ban` directly, if there is any